modules/mobile/app/controllers/mobile/v0/users_controller.rb

```diff
@@ -3,14 +3,19 @@
 module Mobile
   module V0
     class UsersController < ApplicationController
-      after_action :pre_cache_resources, only: :show
-      after_action :handle_vet360_id, only: :show
+      after_action :pre_cache_resources, only: %i[show login]
+      after_action :handle_vet360_id, only: %i[show login]
 
       def show
         map_logingov_to_idme
         render json: Mobile::V0::UserSerializer.new(@current_user, user_accessible_services.authorized, options)
       end
 
+      # called by the mobile app after successful login. performs login-related actions
+      def login
```

Comment:
I agree that login seems like a misrepresentation of what its actually doing but can't think of a great alternative. maybe `post-login` or `precache-and-linking`. or maybe we add a parameter to the show endpoint (v2) to enable or disable those actions? that may make more sense since it solves the backwards compatibility issue I mentioned in another comment and doesn't have the awkwardness of this after action login endpoint.

Reply:
Yeah, I considered post-login but thought it would be confusing to have `POST post-login` and that people would just mean it's a POST login request. I don't want to name it anything specific to the functionality because that may change over time. Ultimately, this should just be a place where we can add any post login actions. 

I'd rather not keep this in the user endpoint because the user show is used at times other than login, which can result in some inefficiency. Also, this is a GET and generally you don't want GETs to have side effects. So I'd prefer this general direction over continuing the status quo, but I'm still torn about naming conventions (as usual).

Reply:
makes sense. maybe `after-login`?

Reply:
`logged-in` or like Andrew said, `after-login`?

Reply:
Changed back to logged-in. Thanks for the feedback.